apps/redi-talent-pool/src/components/organisms/Avatar.tsx

```diff
@@ -201,11 +198,11 @@ const AvatarEditable = ({
       )) as BlobPart
       const croppedImgFile = new File([croppedImage], imageFileName)
 
-      nextFn(croppedImgFile)
+      nextFnRef.current(croppedImgFile)
     } catch (e) {
       console.error(e)
     }
-  }, [imageSrc, imageFileName, croppedAreaPixels, nextFn])
+  }, [imageSrc, imageFileName, croppedAreaPixels, nextFnRef])
```

Comment:
I think by definition you don't need to include refs in hook dependency arrays. They're guaranteed to be stable, so including a ref should have zero effect.

Reply:
It must be the eslint plugin automatically insterting (whenever there's a dependency no matter what it is). Removed now.